packages/edit-site/src/components/style-book/index.js

```diff
@@ -203,6 +203,22 @@ function StyleBook( {
 		[ examples ]
 	);
 
+	const examplesForSinglePageUse = [];
+	const landingCategoryExamples = getExamplesByCategory(
+		{ slug: 'landing' },
+		examples
+	);
+	examplesForSinglePageUse.push( ...landingCategoryExamples.examples );
+	const otherExamples = examples.filter( ( example ) => {
+		return (
+			example.category !== 'landing' &&
+			! landingCategoryExamples.examples.find(
+				( landingExample ) => landingExample.name === example.name
```

Comment:
Nice work filtering out duplicates on the single page 💯

Reply:
Should we memoize the collection of these deduped single page examples?